docs/topics/api.rst

```diff
@@ -102,6 +106,9 @@ Sign-In
         A list of the user's pre-saved `Solitude payment method`_ objects.
         If the user hasn't saved any payment methods yet, this will be an
         empty list.
+    :>json string csrf_token: `Django CSRF`_ token string to protect against
+        cross site request forgery. You must include this in all subsequent
+        requests using the ``X-CSRFToken`` request header.
```

Comment:
Does the token get persisted on the server so it can check? Perhaps I'm confusing with the CSRF in zamboni and can't remember the innards of how CSRF works.